src/cli/commands/object/data.js

```diff
@@ -0,0 +1,41 @@
+'use strict'
+
+const Command = require('ronin').Command
+const utils = require('../../utils')
+const bs58 = require('bs58')
+const debug = require('debug')
+const log = debug('cli:object')
+log.error = debug('cli:object:error')
+
+module.exports = Command.extend({
+  desc: 'Outputs the raw bytes in an IPFS object',
+
+  options: {},
+
+  run: (key) => {
+    var ipfs = utils.getIPFS()
+
+    if (!key) {
+      throw new Error("Argument 'key' is required")
+    }
+
+    const mh = utils.isDaemonOn()
+      ? key
+      : new Buffer(bs58.decode(key))
+
+    ipfs.object.data(mh, (err, data) => {
+      if (err) {
+        log.error(err)
+        throw err
+      }
+
+      if (data instanceof Buffer) {
+        console.log(data.toString())
+        return
```

Comment:
`js-ipfs-api`'s `object.data` accepts multihashes as a string, core needs the `Buffer` instead. 

`js-ipfs-api` returns a `stream`, core returns a `Buffer`